ssn/ssn_separated/ssn.ttl

```diff
@@ -670,9 +658,9 @@ ssn:Observation rdf:type owl:Class ;
                                   owl:minCardinality "0"^^xsd:nonNegativeInteger
                                 ] ,
                                 [ rdf:type owl:Restriction ;
-                                  owl:onProperty ssn:featureOfInterest ;
+                                  owl:onProperty sosa:featureOfInterest ;
```

Comment:
Maxime, let us make changes to the code after our discussions and not before them. This is not a race for who gets most commits accepted for the sake of github statistics. @arminhaller @dr-shorthair